README.md

```diff
@@ -157,7 +157,8 @@ docs-worker-pool contains various triggers for release to higher environments. C
 
 - Create release tags. We currently follow [semver](https://semver.org/) standards.
 - If you don't have push access, open an issue or otherwise contact a contributor with administrator privileges.
-- Verify that the deploy-production-ec2 workflow executed successfully for both job runs across both production instances.
+- Create a release associated with the tag using the GitHub UI or by running `gh release create`. 
```

Comment:
might be an extra space at end of line, based [on error](https://github.com/mongodb/docs-worker-pool/actions/runs/7631054222/job/20788127472?pr=968)